### DISCUSSION TITLE
Is there a way to pin folders to the 1st plane instead of the 2nd plane in Windows Explorer?

### DISCUSSION THREAD

**Discussion by Donglin216** [valinet/ExplorerPatcher#3433] [Q&A]
Hi so I just did a complete reset of my PC, and was readding stuff and setting stuff back up. I was just wondering if explorer patcher had a setting that enabled pinning folders to the 1st plane (Where Home/OneDrive/Gallery is located) instead of the 2nd in windows explorer?

I was able to do it on my old setup but don't remember if I got it here or from somewhere else.

**Comment by pyrates999** [comment]
what do you mean by 1st plane and 2nd plane?

**Reply by Donglin216** [reply]
> what do you mean by 1st plane and 2nd plane?

The place where the Home and Gallery Tab are the 1st plane, and the place where Desktop/Documents/Downloads/etc... is the 2nd plane.

Or maybe the Navigation Pane, sorry never used the new version much

**Comment by pyrates999** [comment]
can you show a screen shot of what you mean?
